crates/cheatnet/src/cheatcodes/elect.rs

```diff
@@ -0,0 +1,17 @@
+use crate::CheatnetState;
+use starknet_api::core::ContractAddress;
+
+impl CheatnetState {
```

Comment:
Should we maybe add a target for this cheatcode too? 🤔

Reply:
Sure, but let's do it in another PR 👍